src/app/modules/main/wallet_section/networks/view.nim

```diff
@@ -94,6 +94,9 @@ QtObject:
   proc getNetworkShortNames*(self: View, preferredNetworks: string): string {.slot.} =
     return self.flatNetworks.getNetworkShortNames(preferredNetworks, self.areTestNetworksEnabled)
 
+  proc getNetworkName*(self: View, chainId: int): string {.slot.} =
+    return self.flatNetworks.getNetworkName(chainId)
```

Comment:
Is method like this needed to be exposed from the backend? If flat networks model is exposed to the ui, it's just one liner to fetch value from the model (`ModelUtils.getByKey`)